docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2024 Kaushlendra Pratap <kaushlendra-pratap.singh@siemens.com>
+# SPDX-License-Identifier: GPL-2.0-only
+services:
+  licensedb:
+    container_name: licensedb
+    build: 
+      context: .
+    environment:
+      DB_PASSWORD: fossy
+      DB_HOST: postgres
+      DB_PORT: 5432
+      DB_USER: fossy
+      DB_NAME: fossology
+      POPULATE_DB: true
+      GIN_MODE: debug
+    ports:
+      - 8080:8080
+    depends_on:
+      postgres:
+        condition: service_healthy
+    restart: always
+    networks:
+      - licensedb-network
+  postgres:
+    container_name: postgres
+    image: postgres:16
+    environment:
+      POSTGRES_PASSWORD: fossy
+      POSTGRES_USER: fossy
+      POSTGRES_DB: fossology
+    ports:
+      - "5432:5432"
```

Comment:
It is not a good idea to expose the database port by default. Please comment the lines.